.github/workflows/setup_template_retail_react_app_manual.yml

```diff
@@ -0,0 +1,72 @@
+name: SalesforceCommerceCloud/pwa-kit/setup_template_retail_react_app
+on:
+  workflow_dispatch:
+
+jobs:
+  setup-template-retail-react-app:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Initialize
+        id: initialize
+        shell: bash
+        run: |
+          set -e
+          echo "PROJECT_SLUG=scaffold-pwa" >> $GITHUB_ENV 
+          branch_name="${GITHUB_REF##*/}"
+          # Trim using cut as MRT target name is limited to 19 chars
+          mrt_target_name=$(echo "$branch_name" | cut -c 1-19)
+          echo "MRT_TARGET_NAME=$mrt_target_name" >> $GITHUB_ENV
+
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        id: setup_node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: "npm"
+
+      - name: Setup PWA dependencies
+        id: setup_pwa_dependencies
+        run: |-
+          set -e
+          sudo apt-get update -yq && sudo apt-get install --no-install-recommends python2 python3-pip time -yq
+          sudo pip install -U pip setuptools
+          sudo pip install awscli==1.18.85 datadog==0.40.1
+          node ./scripts/gtime.js monorepo_install npm ci
+          npm ci
+        shell: bash
+
+      - name: Create MRT credentials file
+        id: create_mrt_credentials
+        uses: "./.github/actions/create_mrt"
+        with:
+          mobify_user: ${{ secrets.MOBIFY_CLIENT_USER }}
+          mobify_api_key: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
+
+      - name: Create MRT target
+        id: create_mrt_target
+        uses: "./.github/actions/create_mrt_target"
+        with:
+          project_name: ${{ env.PROJECT_SLUG }}
+          target_name: ${{ env.MRT_TARGET_NAME }}
+          mobify_api_key: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
+
+      - name: Push bundle to MRT
+        id: push_bundle
+        uses: "./.github/actions/push_to_mrt"
+        with:
+          CWD: "packages/template-retail-react-app"
+          TARGET: ${{ env.MRT_TARGET_NAME }}
+          FLAGS: --wait
+
+      - name: Add proxy config on MRT
+        id: add_proxy_configs
+        uses: "./.github/actions/add_proxy_configs"
+        with:
+          project_slug: ${{ env.PROJECT_SLUG }}
+          target_slug: ${{ env.MRT_TARGET_NAME }}
+          config_file: "./packages/template-retail-react-app/config/default.js"
+          mobify_api_key: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
```

Comment:
Is there a reason that we are separating the creation of the target and the setting of the proxies? This would make for a long deployment as setting proxies causes a longer deployment as it goes to the edge. 

The MRT API says that the [create target API](https://developer.salesforce.com/docs/commerce/de-de/pwa-kit-managed-runtime/references/managed-runtime-api?meta=type%3AAPITargetV2Create) can be passed proxies. Can we use that?

Reply:
Fixed it